microsoft/surface/common/kernel/default.nix

```diff
@@ -5,7 +5,7 @@ let
 
 in {
   imports = [
-    ./linux-6.10.x
+    ./linux-surface
```

Comment:
(non-blocking — this shouldn't block merging the code)

I kind-of like having the kernel version in the directory name, as an obvious pointer of when a reasonable bump of the kernel version happens.